src/components/Favicons.astro

```diff
@@ -1,7 +1,7 @@
 ---
-import favIcon from '~/assets/favicons/favicon.ico';
-import favIconSvg from '~/assets/favicons/favicon.svg';
-import appleTouchIcon from '~/assets/favicons/apple-touch-icon.png';
+import favIcon from '~/assets/favicons/favicon-munch.ico';
+import favIconSvg from '~/assets/favicons/favicon-munch.svg';
```

Comment:
This may be my mistake for not letting you know, but for the icon in the tab, we should just use the "M" from Munch Industries.

Reply:
Is there an image already made for that?

Reply:
No, I think you'll have to take it from the Figma somehow

Reply:
RIP. Ok, I can make something in GIMP

Reply:
Fixed the asset to show only the M. I could not figure out how to get it to switch between black and white for light mode and dark mode.